Untitled.ipynb

```diff
@@ -0,0 +1,116 @@
+{
```

Comment:
Let's not push a notebook called Untitled.ipynb to the top-level directory.  If this has some tests, let's put it under tests/ with a suitable name.

Reply:
Whoops! Thanks for catching!